specification/common/transactions/register/RegisterNewDelegateTransaction.yaml

```diff
@@ -0,0 +1,28 @@
+allOf:
+  - $ref: "./RegisterTransactionBase.yaml"
+  - type: object
+    required:
+      - asset
+      - recipientId
+      - type
+      - senderId
+    properties:
+      asset:
+        $ref: "../assets/VoteForDelegateAsset.yaml"
+      type:
+        type: integer
+        minimum: 2
+        maximum: 2
+        example: 2
+        description: Should be always equal to `2`
+example:
```

Comment:
You can remove this object. Swagger will generate an example automatically by using `example` prop of each property in `schema.properties`

Reply:
but it does not

![image](https://user-images.githubusercontent.com/30230027/203876680-fa522615-def0-49da-88a0-7ce21344192e.png)

Swagger somehow missed the asset part

Reply:
Hm, I see. Then keep it.